src/iexamine.cpp

```diff
@@ -704,7 +704,7 @@ void iexamine::cardreader( player &p, const tripoint &examp )
 {
     bool open = false;
     itype_id card_type = ( g->m.ter( examp ) == t_card_science ? "id_science" :
-                           "id_military" );
+                           g->m.ter( examp ) == t_card_military ? "id_military" : "id_industrial" );
```

Comment:
Won't this change break science card reader functionality?

Reply:
I hope not. This was meant to expand the switch between different card types. Will double check ASAP.

Reply:
A quick test showed that it doesn't break neither the science nor military card ID readers.